storage/sql/sql.go

```diff
@@ -55,6 +56,8 @@ func isRetryableSerializationFailure(err error) bool {
 	return false
 }
 
+const maxRetries = 2
```

Comment:
I thought we already added retries in a previous PR.

Reply:
(rebased away; but #1356 hasn't gone in yet..)